ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -451,3 +474,57 @@ func copyFile(src, dst string) error {
 	}
 	return os.Chmod(dst, srcinfo.Mode())
 }
+
+func findArchManifest(ctx context.Context, imageRef string, pullSecret []byte, osToFind string, archToFind string) (manifestImageRef string, err error) {
```

Comment:
should this be named `findImageRefByArch`? can we comment above what it does?

Reply:
Yeah that's a beter name for it. It's been updated.